src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx

```diff
@@ -93,16 +101,16 @@ class EduBenefitsApp extends React.Component {
       <div className="row">
         {devPanel}
         <Element name="topScrollElement"/>
-        <div className="medium-4 columns show-for-medium-up">
-          <Nav
-              data={data}
-              pages={pageState}
-              chapters={chapters}
-              currentUrl={currentLocation.pathname}/>
+        <div className="medium-8 columns">
+          {
+            // Only render SegmentedProgressBar if we have a current chapter
+            // Could move this into SegmentedProgressBar
+            currentIndex ? <SegmentedProgressBar total={chapters.length} current={currentIndex}/> : null
```

Comment:
Hiding this when we don't have a chapter will probably work, but the actual logic is that we don't want this to show up on the introduction and submit-message pages. You could check for that specifically, which would make what's happening clearer.